script.js

```diff
@@ -21,4 +21,176 @@ navListEl.forEach((element) => {
+}
+
+const modal = document.getElementById('modal1');
+
+for (let j = 0; j < projects.length; j += 1) {
+  const seeBtn = document.querySelectorAll(`[id=btn-${j}]`);
+  seeBtn.forEach((bt) => {
+    bt.addEventListener('click', () => {
+      modal.style.display = 'block';
+      modal.innerHTML = `
+      <div class="popup-container block-popup">
+      <div id="popup-modal" class="pop-content">
+        <div class="headline-popup">
+          <h1>${projects[j].name}</h1>
+          <a href="#" role="button" onclick="closePopup()">
+            <div class="popup-clsbtn">x</div>
+          </a>
+        </div>
+      </div>
+      <ul class="small-tags">
+        <li class="canopy-sm-tag">${projects[j].des[0]}</li>
+        <li>
+          <div class="round-popup"></div>
+        </li>
+        <li class="second-sm-tag">${projects[j].des[1]}</li>
+        <li>
+          <div class="round-popup"></div>
+        </li>
+        <li class="second-sm-tag">${projects[j].des[2]}</li>
+      </ul>
+      <div class="popup-image-container">
+        <img src="${projects[j].img}" alt="pop-image" />
+      </div>
+      <div class="buttom-content-container">
+        <div class="paragraph-description">
+          <p class="paragraph-description-desktop">
+            ${projects[j].desc1}
+          </p>
+        </div>
+        <div class="action-content">
+          <div class="popup-tags">
+            <ul class="first">
+            <li>${projects[j].technologies[0]}</li>
+            <li>${projects[j].technologies[1]}</li>
+            <li>${projects[j].technologies[2]}</li>
+            </ul>
+            <ul class="second">
+            <li>${projects[j].technologies1[0]}</li>
+            <li>${projects[j].technologies1[1]}</li>
+            <li>${projects[j].technologies1[2]}</li>
+            </ul>
+          </div>
+          <div class="divider-popup">
+            <img src="./images/Vector 4.png" alt="Divider popup" />
+          </div>
+          <div class="popup-button-action">
+            <button type="submit">
+              <div class="source-code">
+                <a href="">See live </a>
+              </div>
+              <div class="deploy-icon">
+                <img src="./images/deploy.svg" alt="deployment icon" class="deploy-icon-popup" />
+              </div>
+            </button>
+            <button class="submit-github">
+              <div class="live-version">
+                <a href="">See Source </a>
+              </div>
+              <div class="github-img">
+                <img src="./images/github-popup.svg" alt="github icon" class="github-icon-popup" />
+              </div>
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
+      `;
+
+      const closeBtn = document.querySelector('.popup-clsbtn');
+      closeBtn.addEventListener('click', () => {
+        const modal = document.querySelector('.modal');
+        modal.style.display = 'none';
+      });
+    });
+  });
+}
```

Comment:
You have done a good job implementing the popup interactions and currently, I think your popup looks good on mobile👍. but just as requested by the previous reviewers, the popup on the desktop looks different from that of the design and it is a requirement for this milestone that we stick to the design even as we make our pages responsive😉😉

your popup
![Screenshot from 2023-02-02 10-20-56](https://user-images.githubusercontent.com/64643995/216284073-1754355f-c922-481f-97bf-ad8f5ca54728.png)

design
![image](https://user-images.githubusercontent.com/64643995/216284282-fc40d11a-9c41-4074-99e4-36637f4302ed.png)
I suggest you take note of the height and width of the image and the spacings of each components as well😉

kindly observe the differences and make the necessary adjustments🤗🤗🤗.